### DISCUSSION TITLE
Does the OGC API Features support GeoJSON as output format when using blob mode in deegree?

### DISCUSSION THREAD

**Discussion by Kate-Lyndegaard** [deegree/deegree-ogcapi#57] [Q&A] [answered]
Hi,

Does the OGC API Features also support GeoJSON as output format when using the blob mode in deegree?

Kind regards,
Kate

**Comment by tfr42** [comment, marked as answer]
Yes, all supported backends ([feature stores](https://github.com/deegree/deegree-ogcapi/blob/main/deegree-ogcapi-documentation/src/main/asciidoc/configuration.adoc#feature-store-configuration)) are supported, that includes the [table-driven and BLOB mapping](https://download.deegree.org/documentation/current/html/#anchor-configuration-sqlfeaturestore).
The [example workspace](https://github.com/deegree/deegree-workspaces/tree/main/ogcapi-workspace) contains a `SQLFeatureStore` configuration which can be used as a starting point. To import data use the [GmlLoader](https://download.deegree.org/documentation/current/html/#deegree-gml-tools) which also supports the BLOB mapping.

**Reply by Kate-Lyndegaard** [reply]
Hi @tfr42,

Thank you for your reply. That is great news.

Kind regards,
Kate
